lib/generators/jsonapi/setup/setup_generator.rb

```diff
@@ -0,0 +1,85 @@
+module Jsonapi
+  module Generators
+    class SetupGenerator < Rails::Generators::Base
+      desc 'Setup base gems and configuration'
+      source_root File.expand_path('../../templates', __FILE__)
+
+      def add_gems
+        insert_into_file 'Gemfile', global_gems, before: first_gem
+        insert_into_file 'Gemfile', test_gems, after: "group :test do\n"
```

Comment:
Is 

```
gem_group :test do
  gem "rspec-rails"
end
```

smart enough to find the group if it exists and add the line?

http://guides.rubyonrails.org/generators.html#gem-group